src/main/kotlin/assimp/format/obj/ObjFileParser.kt

```diff
@@ -256,16 +256,18 @@ class ObjFileParser(private val file: IOStream, val ioSystem: IOSystem) {
         if (words.size < 2) throw Error("File name of the material is absent.")
 
         // get the name of the mat file with spaces
-        var filename = ObjTools.getNameWithSpace(words, 1)
+        val filename = ObjTools.getNameWithSpace(words, 1)
 
-        val pFile = file.parentPath() + "/" + filename //windows can just suck it
+        val pFile = file.parentPath + ioSystem.osSeparator + filename //windows can just suck it
         println(pFile)
 
         if (!ioSystem.exists(pFile)) {
```

Comment:
keeping this one?

Reply:
woops, I remove the comment. I think this is what you meant :wink:

Reply:
sorry, was talking about the iosystem.exists three lines below. though the comment too lol